app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordRepository.kt

```diff
@@ -0,0 +1,49 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import org.simple.clinic.storage.Timestamps
+import org.simple.clinic.util.UtcClock
+import org.simple.clinic.uuid.UuidGenerator
+import java.time.Instant
+import java.util.UUID
+import javax.inject.Inject
+
+class TeleconsultRecordRepository @Inject constructor(
+    private val uuidGenerator: UuidGenerator,
+    private val teleconsultRecordDao: TeleconsultRecord.RoomDao,
+    private val teleconsultRecordWithPrescribedDrugsDao: TeleconsultRecordWithPrescribedDrugs.RoomDao,
+    private val utcClock: UtcClock
+) {
+
+  fun getTeleconsultRecordWithPrescribedDrugs(teleconsultRecordId: UUID): TeleconsultRecordWithPrescribedDrugs? {
+    return teleconsultRecordWithPrescribedDrugsDao.getPrescribedUuidForTeleconsultRecordUuid(teleconsultRecordId)
+  }
+
+  fun createTeleconsultRecord(
```

Comment:
This is the record on the nurse side, right? Maybe we should consider naming this method differently to expose this detail, like `createTeleconsultRecordForRequester`.

Reply:
No, this is the record on MO side. If there is no teleconsult record already we will create this, or else we will use the teleconsult record with prescribed drugs to prefill the screen and use that.

Reply:
Oh yes, my bad. The point I wanted to bring up was just that maybe we should have different named parameters for constructing the teleconsult record differently on either side.

Reply:
Yeah, good point. Renamed it to `createTeleconsultRecordForMO`

Reply:
We can do this in a later PR, but we really should try to avoid abbreviations. Let's just have a longer method name and expand `MO` where it is being used.